aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -32,6 +32,14 @@ Tensor dequantize_quant(const Tensor& self) {
   return get_qtensorimpl(self)->quantizer()->dequantize(self);
 }
 
+std::vector<Tensor> dequantize_tensors_quant(TensorList tensors) {
```

Comment:
Don't return a std::vector, return a TensorList instead (TensorList == torch::List<Tensor>). std::vector requires a copy behind the scenes.

Reply:
I think that is not the type generated by native_functions.yaml, I looked up another function that returns Tensor[] and it says:
```
795:std::vector<Tensor> meshgrid(TensorList tensors); // {"schema": "aten::meshgrid(Tensor[] tensors) -> Tensor[]", "compound": "true"}
```